Objects/unicodeobject.c

```diff
@@ -8511,15 +8515,19 @@ encoding_map_lookup(Py_UCS4 c, PyObject *mapping)
    can't be found, Py_None is returned (or NULL, if another
    error occurred). */
 static PyObject *
-charmapencode_lookup(Py_UCS4 c, PyObject *mapping)
+charmapencode_lookup(Py_UCS4 c, PyObject *mapping, unsigned char *replace)
```

Comment:
Can you explain in the comment when replace is set?

Reply:
Done.